service/worker/scheduler/workflow.go

```diff
@@ -112,9 +114,18 @@ var (
 		},
 	}
 
-	defaultCatchupWindowParams = catchupWindowParams{
-		Default: 60 * time.Second,
-		Min:     10 * time.Second,
+	// We put a handful of options in a static value and use it as a MutableSideEffect within
+	// the workflow so that we can change them without breaking existing executions or having
+	// to use versioning.
+	currentTweakablePolicies = tweakablePolicies{
```

Comment:
also, "change" could mean dynamic config instead of this constant. I don't think we need to promote any of these to dynamic config right now, but it's an option for later.

Reply:
sg